pangeo_forge_recipes/writers.py

```diff
@@ -148,7 +148,14 @@ def write_combined_reference(
     else:
         raise NotImplementedError(f"{file_ext = } not supported.")
 
-    return full_target
+    # Return an fsspec mapper that can be read with Xarray
+    return ReferenceFileSystem(
+        outpath,
+        remote_protocol=remote_protocol,
+        remote_options={"anon": True},
+        target_protocol="file",
```

Comment:
Comments on each of these:

- `remote_protocol`: IIUC, this is the protocol the data lives on, which is not what this variable has been assigned to above. This variable is assigned to to the protocol that the reference is written to.
- `remote_options`: We don't know for sure that these are the options users will need to access the remote data. This needs to be configurable as a kwarg. Possibly `anon:True` is a sensible default for that kwarg.
- `target_protocol`: IIUC, this is the protocol on which the reference file exists. So it is the the value that the `remote_protocol` variable is currently assigned to.